### PR TITLE
Portfolio Deployment

### PR DESCRIPTION
**Pull request by Hamdullah-Andar**
## Deploy My Portfolio

- Deploy Portfolio on Github
- Add the live demo link to the README file

Thank you for the review

### CODE REVIEW

**Review by nahnahdev — COMMENTED** [review]
Hi @Hamdullah-Andar, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is:
- [x]  Missing some headers in your README file, in the screenshot below, you can see that after the `Tech Stack`, you're supposed to have the `Key Features `section. But you don't have that section in your README file. 
- [x] You also don't have the Future Features section in your project this comes after the `Authors` section :wink: Please ensure that you add at least one `Future Feature `to your README file :wink: 
- [x] So kindly add these as well so your project can be reviewed :wink:. This is necessary because every section of the `Microverse README template `is compulsory with the exception of the `FAQ's section `:100: 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -72,6 +72,28 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (OyePriscilla) [inline README.md]: - [x] You've done well by adding key feature as requested, but please kindly remove this template that you mistakenly added alongside (I guess)  so as to make your README file look professional. 👍 

> **Note: The same is applicable to `line 91 (Live Demo)` and `line 174 (Future Feature)`**.

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Hamdullah-Andar 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
App deployed as expected 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [optional] Please kindly ensure that your MIT license link in README file is not broken. You may need to create a LICENSE file in your root folder and perhaps personalize this [template](https://opensource.org/licenses/MIT). It's a short and simple permissive license with conditions only requiring preservation of copyright and license notices. ℹ️ 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -72,6 +72,26 @@ After you're finished please remove all the comments and instructions!
...
```
Comment (IngDavidTM) [inline README.md]: - [x] While you did a good job deploying and customizing your README file :+1:, as required by the previous reviewer kindly make sure to remove the comments like `Describe between 1-3 key features of the application.` from your readme :+1:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Hamdullah-Andar  🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**